tests/robot-tests/tests/admin/bau/comments.robot

```diff
@@ -0,0 +1,200 @@
+    user adds content to accordion section text block    ${CONTENT_BLOCK_TITLE_1}    1    ${CONTENT_BLOCK_BODY_1}
+    ...    css:#releaseMainContent
+
+Switch to bau1 to add review comments
+    [Tags]    HappyPath
+    user changes to bau1
+    user selects theme and topic from admin dashboard    %{TEST_THEME_NAME}    %{TEST_TOPIC_NAME}
+    user opens publication on the admin dashboard    ${PUBLICATION_NAME}
+    ${accordion}=    user gets accordion section content element    ${PUBLICATION_NAME}
+    user opens details dropdown    ${RELEASE_NAME}    ${accordion}
+    ${details}=    user gets details content element    ${RELEASE_NAME} (not Live)    ${accordion}
+    user waits until parent contains element    ${details}    xpath:.//a[text()="Edit this release"]
+    user clicks link    Edit this release
+
+Navigate to content section as bau1
+    [Tags]    HappyPath
+    user clicks link    Content
+    user closes Set Page View box
+
+Add review comment for first content block
+    [Tags]    HappyPath
+    user scrolls to element    testid:accordionSection
+    user clicks element    xpath://button[text()="First content section"]
+
+    user scrolls to element    testid:Expand Details Section Add / View comments (0 unresolved)
+    user clicks element    testid:Expand Details Section Add / View comments (0 unresolved)
+
+    # avoid set page view box getting in the way
+    user scrolls down    400
+    user presses keys    This section needs fixing    testid:comment-textarea
+    user clicks button    Add comment
+
+    user waits until page contains element
+    ...    //*[@data-testid="comment-content" and text()="This section needs fixing"]    5
+    user clicks element    testid:Expand Details Section Add / View comments (1 unresolved)
+
+Switch to analyst1 and navigate to release
+    [Tags]    HappyPath
+    user changes to analyst1
+    user selects theme and topic from admin dashboard    %{TEST_THEME_NAME}    %{TEST_TOPIC_NAME}
+    user opens publication on the admin dashboard    ${PUBLICATION_NAME}
+    ${accordion}=    user gets accordion section content element    ${PUBLICATION_NAME}
+    user opens details dropdown    ${RELEASE_NAME}    ${accordion}
+    ${details}=    user gets details content element    ${RELEASE_NAME} (not Live)    ${accordion}
+    user waits until parent contains element    ${details}    xpath:.//a[text()="Edit this release"]
+    user clicks link    Edit this release
+
+Navigate to content section to resolve comments
+    [Tags]    HappyPath
+    user clicks link    Content
+    user waits until h1 is visible    ${PUBLICATION_NAME}
+
+Resolve comments
+    [Tags]    HappyPath
+    user closes Set Page View box
+    user scrolls to element    testid:accordionSection
+    user clicks element    xpath://button[text()="First content section"]
+    user clicks element    //*[@data-testid="Expand Details Section Add / View comments (1 unresolved)"]
```

Comment:
Is the `testid:`  custom locator working now in all instances? If so can use `testid:Expand Details Section Add / View comments (1 unresolved)` here. There are a couple of other instances of data-testids being selected by xpath that could potentially use `testid:` too.